### ISSUE TITLE
Device network exceptions

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#16642]
![image](https://github.com/space-wizards/space-station-14/assets/31366439/b16b6a8b-837a-4fe2-b55d-0f974c32fae6)

@juliangiebel

**Comment by juliangiebel** [comment]
I think I just forgot to set do not throw to true on these resolves:
```cs
var sourceAddress = Resolve(sourceUid, ref sourceNetworkComponent) ? sourceNetworkComponent.Address : "";
var sinkAddress = Resolve(sinkUid, ref sinkNetworkComponent) ? sinkNetworkComponent.Address : "";
```

**Comment by juliangiebel** [comment]
Do those errors still appear now that this PR was merged? #16675

**Comment by metalgearsloth** [comment]
> Do those errors still appear now that this PR was merged? #16675

yeah

**Comment by metalgearsloth** [comment]
![image](https://github.com/space-wizards/space-station-14/assets/31366439/9c82076d-4d2d-4ffa-9daa-ca1aa5432ad6)
 15 minutes ago
